libs/domain/cart/src/services/cart.providers.ts

```diff
@@ -37,4 +37,8 @@ export const cartProviders: Provider[] = [
   CartTotalsProvider,
   ...cartNormalizer,
   ...cartsNormalizer,
+  // {
```

Comment:
👀

Reply:
I commented out it because of CORS issue